include/GafferSceneUI/Private/Inspector.h

```diff
@@ -352,6 +357,7 @@ class GAFFERSCENEUI_API Inspector::Result : public IECore::RefCounted
 
 		EditFunctionOrFailure m_editFunction;
 		std::string m_editWarning;
+		std::string m_fallbackDescription;
```

Comment:
This would probably also be a bit clearer grouped under `m_sourceType`.

Reply:
Reordered in the same fixup as above: f59c3f9.